### DISCUSSION TITLE
Nouvel utilisateur - guide composteur

### DISCUSSION THREAD

**Discussion by thojongle** [compostri/pom-e#14] [General]
Bonjour 
Nouvel utilisateur en tant que référent d'un site de compost, je souhaite contribuer à quelques améliorations et apporter les idées liées à la pratique terrain.

Peut être que le rôle de guide composteur, apportera aussi des suggestions dans la création d'un suivi des besoins des utilisateurs et nouveaux utilisateurs.

Un exemple, j'ai rencontré une frustrée suite à l'absence de permanence et pas de réponses via les tentatives de contact que la plateforme.

Une personne externe tente un contact via la plateforme, auprès du référent mais trois mois après n'a pas de réponse.
Est-ce qu'un système de relance ou l'envoi de la requête vers le guide composteur de la zone et le maître composteur, peut assurer une réponse ?

Il est possible que la réponse soit réalisée de façon physique et ainsi, permettre au référent de valider le statut "répondu".

Bonne journée

**Comment by ArnaudBan** [comment]
Bonjour,
Il arrive qu'aucun référent sur un site de compostage accepte de recevoir les mails ou ne soit inscrit pour recevoir les mails. Chaque référent gère ce paramètre dans son profile. Je commencerai par vérifier cela.
Si aucun référent n'est associé a une site de compostage c'est le maitre composteur que reçoit les demandes.

Il est probable qu'ici le problème soit humain et que soit le/les référents est bloqué les notifications, soit les référents ou maitre composteur n'ont pas pris le temps de répondre.

Si vous voulez on peut regarder ensemble dans un exemple précis a qui les demandes sont envoyé pour voir si il y a un probléme sur la plateforme ( normalement il doit toujours y avoir personne qui recoit les demandes )

**Reply by thotanguy** [reply]
Merci Arnaud,
Je pense aussi que dans ce cas précis le problème est humain.
